src/commonNative/kotlin/com/epam/drill/util/StringProperty.kt

```diff
@@ -28,6 +28,10 @@ class StringProperty(val map: Map<String, String>) : NamedValueDecoder() {
         return map.getValue(tag).toBoolean()
     }
 
+    override fun decodeTaggedLong(tag: String): Long {
```

Comment:
There are no usages of this method in the request.

Reply:
This is required for parsing the agent's arguments. Previously there were only strings and Boolean, now there is a long for duration